src/App.js

```diff
@@ -14,11 +14,95 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    nameOfProduct: 'Selected good : ',
```

Comment:
```suggestion
    nameOfProduct: '',
```
store here only the name of the goods